tests/integration/kubernetes/run_kubernetes_nv_tests.sh

```diff
@@ -13,12 +13,18 @@ source "${kubernetes_dir}/../../common.bash"
 
 # Enable NVRC trace logging for NVIDIA GPU runtime
 enable_nvrc_trace() {
```

Comment:
minor: function can be entered by multiple test cases, need to ensure the sed statements are idempotent